ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/event_handlers.schema.yml

```diff
@@ -42,6 +42,70 @@ keys:
           - on-boot
           - on-logging
           - on-startup-config
+          - on-maintenance
+        trigger_on_maintenance:
+          type: dict
+          keys:
```

Comment:
Add a description here like `Settings required for trigger 'on-maintenance'.`

Reply:
added